website/static/js/registrationUtils.js

```diff
@@ -516,6 +534,15 @@ var RegistrationEditor = function(urls, editorId) {
         }
     });
 
+    self.markAsRequired = function() {
+
+        var questions = self.flatQuestions();
+        $.each(questions, function(idx, question) {
```

Comment:
What is the purpose of this method? When do we want to mark all questions as required?